aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -753,6 +754,31 @@ func resourceAviatrixTransitGateway() *schema.Resource {
 					},
 				},
 			},
+			"interface_mapping": {
+				Type:        schema.TypeList,
```

Comment:
Does order matter here? Do you want to avoid duplicates?  You may want to consider TypeSet

Reply:
I need the list ordering for interface mapping and the current interface_mapping works with the example provided in the description.